### DISCUSSION TITLE
Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 70

### DISCUSSION THREAD

**Discussion by RajeshKasture** [appium/.github#4] [Q&A]
Im not able to build my project on katalon studio on iOS platform. 
After installing application on real device the wbdriveragent runner apllication is getting uninstalled.
I am using below software:

MacBook =Macos ventura    version=13.6.3
iPhone 14 version =16.4
Katalon Studio = 9.2.0
Xcode version =15.0.0
Homebrew version = 4.2.7
node version =21.6.1
npm version =10.2.4
appium version = 2.5.1
uiautomator2 version = 3.0.0
version = xcuitest 7.1.0
java version = "21.0.1"

When i try to record a new script on katalon studio for iOS i get something like this:

java.util.concurrent.ExecutionException: org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 70. This usually indicates an issue with the local Xcode setup or WebDriverAgent project configuration or the driver-to-platform version mismatch. Consider setting 'showXcodeLog' capability to true in order to check the Appium server log for build-related error messages.. Make sure you follow the tutorial at https://github.com/appium/appium-xcuitest-driver/blob/master/docs/real-device-config.md. Try to remove the WebDriverAgentRunner application from the device if it is installed and reboot the device.
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'Admins-MacBook-Pro.local', ip: '192.168.1.5', os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '13.6.3', java.version: '17.0.7'
Driver info: driver.version: IOSDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Unable to launch WebDriverAgent because of xcodebuild failure: xcodebuild failed with code 70. This usually indicates an issue with the local Xcode setup or WebDriverAgent project configuration or the driver-to-platform version mismatch. Consider setting 'showXcodeLog' capability to true in order to check the Appium server log for build-related error messages.. Make sure you follow the tutorial at https://github.com/appium/appium-xcuitest-driver/blob/master/docs/real-device-config.md. Try to remove the WebDriverAgentRunner application from the device if it is installed and reboot the device.
    at getResponseForW3CError (/opt/homebrew/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/errors.js:1118:9)
    at asyncHandler (/opt/homebrew/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/protocol.js:491:57)
    at runNextTicks (node:internal/process/task_queues:60:5)
    at processImmediate (node:internal/timers:449:9)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'Admins-MacBook-Pro.local', ip: '192.168.1.5', os.name: 'Mac OS X', os.arch: 'aarch64', os.version: '13.6.3', java.version: '17.0.7'
Driver info: driver.version: IOSDriver

I have checked different xcuitest versions but still getting the same error. Can anyone help on this.

Thanks

**Comment by RajeshKasture** [comment]
Hi Can anyone help on this

Thanks in advance.

**Comment by tatanota** [comment]
私も同じエラーが出たことがあります。

Appiumインストール時にsudoでインストールした際に「xcodebuild failed with code 70.」が表示されました

**Comment by crossterra-agent** [comment]
- **Status**: DONE
- **Role**: orchestrator
- **Step**: 
- **Link**: https://github.com/CrossTerra/monorepo/pull/1912#issuecomment-4034044223
- **Summary**: Developer completed all steps successfully; requested completion review on PR #1912.
